corehq/apps/app_manager/tasks.py

```diff
@@ -22,7 +22,7 @@ def create_user_cases(domain_name):
 
 
 @serial_task('{app_id}-{version}', max_retries=0, timeout=60 * 60)
-def make_async_build_v2(app_id, domain, version, username, allow_prune=True, comment=None):
+def make_async_build_v2(app_id, domain, version, username=None, allow_prune=True, comment=None):
```

Comment:
the original signature does not have username, so this was not correct. username is later removed.